src/main/java/com/three_tech_solutions/slot_app/controllers/interfaces/UserController.java

```diff
@@ -17,6 +17,8 @@ public interface UserController {
     Page<StudentResponse> getUserStudents(
             @PathVariable UUID userId,
             @RequestParam(required = false, defaultValue = "") String filter,
+            @RequestParam(required=false, defaultValue="") String orderBy,
+            @RequestParam(required=false, defaultValue="") String orderDirection,
```

Comment:
Hay que mantener un estándar en el código cuando programamos, acá por ejemplo debería haber espacios que separen el "="

```suggestion
            @RequestParam(required = false, defaultValue = "") String orderBy,
            @RequestParam(required = false, defaultValue = "") String orderDirection,
```